### PR TITLE
Updated the Calculator projects to include the /Z7 compiler flag

### PR DESCRIPTION
**Pull request by sanderl**
### Description of the changes:
- Added the /Z7 compiler flag to the AdditionalOptions compiler configuration

### CODE REVIEW

File: src/Calculator/Calculator.vcxproj [hunk 1]
```diff
@@ -135,7 +135,7 @@
   </ImportGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">
     <ClCompile>
-      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8</AdditionalOptions>
+      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8 /Z7</AdditionalOptions>
```
Comment (mcooley) [inline src/Calculator/Calculator.vcxproj:138]: While we're here, can we add %(AdditionalOptions) for consistency?

File: src/Calculator/Calculator.vcxproj [hunk 2]
```diff
@@ -144,7 +144,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">
     <ClCompile>
-      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8</AdditionalOptions>
+      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8 /Z7</AdditionalOptions>
```
Comment (mcooley) [inline src/Calculator/Calculator.vcxproj:147]: Same here.

File: src/Calculator/Calculator.vcxproj [hunk 3]
```diff
@@ -177,7 +177,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
-      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8</AdditionalOptions>
+      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8 /Z7</AdditionalOptions>
```
Comment (mcooley) [inline src/Calculator/Calculator.vcxproj:180]: And here.

File: src/Calculator/Calculator.vcxproj [hunk 4]
```diff
@@ -198,7 +198,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <ClCompile>
-      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8</AdditionalOptions>
+      <AdditionalOptions>/bigobj /await /std:c++17 /utf-8 /Z7</AdditionalOptions>
```
Comment (mcooley) [inline src/Calculator/Calculator.vcxproj:201]: One more.

**Comment by Rahul-Dixit** [comment]
/cc @sanderl @grochocki @mcooley After merging this pull request, why is the compiler issuing a lot of warnings stating:
```
[warning]cl(0,0): Warning D9025: overriding '/Zi' with '/Z7'
```
See the [Pipeline logs](https://dev.azure.com/ms/calculator/_build/latest?definitionId=57&branchName=master) for more information.

**Comment by sanderl** [comment]
@Rahul-Dixit, this is a warning that is caused because the /Z7 compiler flag is explicitly overriding the default /Zi compiler flag. I will be looking into suppressing this message if we need to.

**Pull request merged**